File: Backup/PlaceYourBets/MenuForm.vb
Public Class MenuForm

    Private Sub enterPredictionsButton_Click(sender As System.Object, e As System.EventArgs) Handles enterPredictionsButton.Click
        FixturesForm.Show()
        Me.Close()
    End Sub


    Private Sub reviewCurrentBet_Click(sender As System.Object, e As System.EventArgs) Handles reviewCurrentBet.Click
        ReviewForm.Show()
        Me.Close()
    End Sub

    Private Sub shutdownButton_Click(sender As System.Object, e As System.EventArgs) Handles shutdownButton.Click
        Application.Exit()
    End Sub

    Private Sub SubmittedPredictionsButton_Click(sender As System.Object, e As System.EventArgs) Handles SubmittedPredictionsButton.Click
        PredictionsSubmitted.Show()
        Me.Close()
    End Sub

    Private Sub PredictionLeagueTableButton_Click(sender As System.Object, e As System.EventArgs) Handles PredictionLeagueTableButton.Click
        PredictionLeagueTableForm.Show()
        Me.Close()
    End Sub
End Class

File: PlaceYourBets.ConvertedToC#/MenuForm.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
using System.Linq;
using System.Xml.Linq;
namespace PlaceYourBets
{
	public partial class MenuForm
	{

		private void enterPredictionsButton_Click(System.Object sender, System.EventArgs e)
		{
			My.MyProject.Forms.FixturesForm.Show();
			this.Close();
		}


		private void reviewCurrentBet_Click(System.Object sender, System.EventArgs e)
		{
			My.MyProject.Forms.ReviewForm.Show();
			this.Close();
		}

		private void shutdownButton_Click(System.Object sender, System.EventArgs e)
		{
			Application.Exit();
		}

		private void SubmittedPredictionsButton_Click(System.Object sender, System.EventArgs e)
		{
			My.MyProject.Forms.PredictionsSubmitted.Show();
			this.Close();
		}

		private void PredictionLeagueTableButton_Click(System.Object sender, System.EventArgs e)
		{
			My.MyProject.Forms.PredictionLeagueTableForm.Show();
			this.Close();
		}
		public MenuForm()
		{
			InitializeComponent();
		}
	}
}

